src/index.html

```diff
@@ -6,7 +6,43 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+
+    <div class="calendar page__calendar">
+
+      <div class="calendar__day calendar__day--start-day--sun"></div>
```

Comment:
Task says that modifler must be on `calendar`, not `calendar__day`.
Task also says: `Add a modifier month-length for the calendar with values 28, 29, 30 and 31`, now you dont have it.